biolink-model.yaml

```diff
@@ -2153,6 +2188,97 @@ slots:
     mappings:
       - rdf:type
 
+  ## Additional Properties for Information Content Entity and Publication Nodes
+
+  doi:
+    is_a: id
+    domain: information content entity
+    range: doi type
+    exact_mappings:
+      - WIKIDATA_PROPERTY:P356
+
+  genre:
```

Comment:
is this a standard term? Seems odd and strangely generic

Why would we not type this the same way we type other instances in the model

Reply:
You mean the namespace **WIKIDATA_PROPERTY**?  Had a dialog with Harold about this. Wikidata is non-standard in its base URI for Wikidata properties. As you know,  the main base URI for Wikidata is `https://www.wikidata.org/wiki/`. "WIKIDATA" is publicly used as the prefix for this in some contexts.  On the other hand, Wikidata Properties have the (rather problematid) base URI `https://www.wikidata.org/wiki/Property:`. Note the trailing colon... If one uses WIKIDATA:P31, it does NOT resolve to the Property P31 ('instance of'), but WIKIDATA:Property:P31 could potentially resolve but breaks BiolinkML parsing.   Defining  **WIKIDATA_PROPERTY** in the local prefixes: specification as `https://www.wikidata.org/wiki/Property:` and using the said prefix for Wikidata Property identifiers  seems to work

Reply:
Sorry... I just realized that you might rather be referring to the node property **genre** in your comment above.  I chose the word 'genre' inspired by one of the links that Melissa gave you (and used in the issue): http://vocabularies.coar-repositories.org/documentation/resource_types/ which used the term. I was going to use  "publication type" but there may have been some practical challenges with using that property name which discouraged me.